packages/components/select/src/useSelect.ts

```diff
@@ -731,6 +731,11 @@ export const useSelect = (props, states: States, ctx) => {
     deleteSelected(event)
   }
 
+  const handlePickUpSelect = (event: Event) => {
```

Comment:
Although I understand what the code is doing — mainly because I read the issue's description —, I couldn't really understand what you mean with "pickUpSelect". 

I suggest changing the naming of this to `handleCaretClick`.

Reply:
Ok, because my English ability is poor, so I will name it like this, your suggestion is good, I think it is ok

Reply:
No problem @smalller, we are all here to help!
Your PR is a fix for an issue that my QA team reported this week after I migrated my project from Element-UI to Element-Plus, so thank you for this!